custom_components/moonraker/sensor.py

```diff
@@ -219,3 +243,17 @@ def _handle_coordinator_update(self) -> None:
             self.coordinator.data
         )
         self.async_write_ha_state()
+
+
+def calculate_pct_job(data) -> float:
+    """Get best estimate of %"""
```

Comment:
I understand that it's "better" than before but it will look weird for a new reader